src/client/Session.cpp

```diff
@@ -48,4 +50,17 @@ void Session::release() {
     }
 }
 
+void Session::toLocal(DataSet &data, int32_t offsetSecs) {
```

Comment:
You don't have to provide a method like that, but you can tell user how to get localTime and get localDateTime in example code

Reply:
I will add example.

Reply:
The interface better move to test code, no need to provide it to the user

Reply:
Use could use or not.